rpc/codec/server.go

```diff
@@ -24,11 +24,17 @@ package codec
 import (
 	"bufio"
 	"bytes"
+	"crypto/tls"
 	"fmt"
 	"io"
 	"net/rpc"
 
+	"github.com/cockroachdb/cockroach/base"
+	cockroach_proto "github.com/cockroachdb/cockroach/proto"
```

Comment:
We (err, @tamird) is looking into splitting up the `proto` package. There is no particular reason to have the protos separated from their usage. For example, the `MVCC` protos should live in `storage/engine`. We might have a base protos package which would make sense to name `roachpb` or `pb`, but hold off on doing any renaming yet until @tamird gets started with the split.